desktop-app/app/components/UrlSearchResults/index.js

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
+import styles from '../Addressinput/style.css';
```

Comment:
@abakermi Do we need this import in this component?

Reply:
![Peek 2020-07-24 12-11](https://user-images.githubusercontent.com/60294727/88376909-fd877680-cda6-11ea-92cb-a714f0d2438e.gif)

Reply:
@manojVivek I have  removed the two lines that you mention, in term of UX do we need to change the color of hover/active color to match the light blue on the left ?

Reply:
@abakermi I think the `white` text color is good enough. Making it blue will emphasize it a little too much?

Reply:
contrast light blue on gray background good also because if you see you have 2 min colors, also the theme overall promote dark mode,